libs/openFrameworks/utils/ofConstants.h

```diff
@@ -188,14 +188,18 @@ enum ofTargetPlatform{
 
 	#include <unistd.h>
 
+  #include "bcm_host.h"
+  #undef countof
 	#ifdef TARGET_LINUX_ARM
-		#ifdef TARGET_RASPBERRY_PI
+		// #ifdef TARGET_RASPBERRY_PI
```

Comment:
this should be kept as it was and the include for bcm_host above removed, since it only belongs to the PI not to general linux. Also the define countof. if it's needed at all should go here not in the general linux section

Reply:
so If I'm trying to compile with TARGET_LINUX, what'd be the right way to stick that countof undefine?